install/Docker/dockerfiles/Dockerfile.accountSession

```diff
@@ -0,0 +1,45 @@
+#(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+#Licensed under the Apache License, Version 2.0 (the "License"); you may
+#not use this file except in compliance with the License. You may obtain
+#a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#License for the specific language governing permissions and limitations
+# under the License.
+
+FROM golang:1.13.8 as build-stage
+
+WORKDIR /account-session/
+
+ADD svc-account-session/go.sum .
+ADD svc-account-session/go.mod .
+
+RUN go mod download
+
+ADD svc-account-session .
+
+RUN go build -i -race .
+COPY install/Docker/dockerfiles/scripts/add-hosts.go .
+RUN go build -o add-hosts add-hosts.go
+
+FROM ubuntu:18.04
+
+RUN groupadd -r -g 2021 odimra
+RUN useradd -s /bin/bash -u 2021 -m -d /home/odimra -r -g odimra odimra
```

Comment:
User ID and Group ID should be configurable by user. Better to take it as args during image build.

Reply:
Added as below:
RUN if [ -z "$ODIMRA_USER_ID" ] || [ -z "$ODIMRA_GROUP_ID" ]; then echo "\n[$(date)] -- ERROR -- ODIMRA_USER_ID or ODIMRA_GROUP_ID is not set\n"; exit 1; fi \
&& groupadd -r -g $ODIMRA_GROUP_ID odimra